src/App.js

```diff
@@ -1,28 +1,11 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
+      <Calculator />
+    </>
```

Comment:
- [ ] Please according to project requirements take into account your App.js component should be class-based

![image](https://user-images.githubusercontent.com/95243996/218743631-3bd4f242-2e79-4fb0-9dad-1b89c6162448.png)

Reply:
Hi @andgarzonmal the project requirement states we use functional components.

![image](https://user-images.githubusercontent.com/116718155/218744803-3280f474-9c1a-4ea4-adf9-9df6530c7cd8.png)

And some students get approval for using functional components. Seems the project requirements have been updated recently

![image](https://user-images.githubusercontent.com/116718155/218745179-2a2871d4-dd32-4604-bfde-ad2a6deec491.png)